### ISSUE TITLE
Install wildfly 26.0.1.Final using Galleon 4.2.8 fails

### ISSUE THREAD

**Issue by mjourdan** [Keyfactor/ansible-ejbca-signserver-playbooks#28]
Deploying ejbca CE with the following command:

```
$ ansible-playbook -i inventory -l ce01 deployCeNode.yml --ask-vault-pass
```

Installation fails for us after the following steps:

```
TASK [ansible-ejbca-wildfly : Download Wildfly 26.0.1.Final and install software] *******************************************************************************************************************************
included: /home/mjourdan/git/ansible-ejbca-signserver-playbooks/ansible_ejbca_signsrv/roles/ansible-ejbca-wildfly/tasks/install.yml for ce01

TASK [ansible-ejbca-wildfly : Install OpenJDK] ******************************************************************************************************************************************************************
ok: [ce01]

TASK [ansible-ejbca-wildfly : Install unzip] ********************************************************************************************************************************************************************
ok: [ce01]

TASK [ansible-ejbca-wildfly : Check if wildfly directory exists] ************************************************************************************************************************************************
ok: [ce01]

TASK [ansible-ejbca-wildfly : Download wildfly 26.0.1.Final tar file] *******************************************************************************************************************************************
skipping: [ce01]

TASK [ansible-ejbca-wildfly : Download wildfly Galleon 4.2.8] ***************************************************************************************************************************************************
changed: [ce01]

TASK [ansible-ejbca-wildfly : Create EJBCA group - wildfly] *****************************************************************************************************************************************************
changed: [ce01]

TASK [ansible-ejbca-wildfly : Create EJBCA user - wildfly] ******************************************************************************************************************************************************
changed: [ce01]

TASK [ansible-ejbca-wildfly : Unarchive downloaded wildfly 26.0.1.Final tar file] *******************************************************************************************************************************
skipping: [ce01]

TASK [ansible-ejbca-wildfly : Unarchive downloaded wildfly Galleon 4.2.8 file] **********************************************************************************************************************************
changed: [ce01]

TASK [ansible-ejbca-wildfly : Install wildfly 26.0.1.Final using Galleon 4.2.8] *********************************************************************************************************************************
fatal: [ce01]: FAILED! => changed=true
  cmd:
  - /opt/galleon-4.2.8.Final/bin/galleon.sh
  - install
  - wildfly:current#26.0.1.Final
  - --dir=/opt/wildfly-26.0.1.Final
  - --default-configs=standalone/standalone.xml
  - --layers=cloud-server,deployment-scanner,ejb,io,jsf,mail,webservices,-jsonb,-jms-activemq,-observability,-jmx-remoting,core-management,core-tools,management
  delta: '0:00:01.229689'
  end: '2022-10-12 09:04:33.652831'
  msg: non-zero return code
  rc: 1
  start: '2022-10-12 09:04:32.423142'
  stderr: ''
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>

PLAY RECAP ******************************************************************************************************************************************************************************************************
ce01                       : ok=28   changed=4    unreachable=0    failed=1    skipped=24   rescued=0    ignored=0
```

## Steps to reproduce

Here is the target host OS:

```
# grep -i pretty /etc/os-release
PRETTY_NAME="Red Hat Enterprise Linux 8.6 (Ootpa)"
```

Our local repository is up-to-date:
```
$ git rev-parse origin/master
1397710bdcdcf630a8815431d32d822a7c51858e
```

The only changes we've made above what's in the repo are basically:
- network settings (host ip address, use of http_proxy)
- some organizational changes in `group_vars/all.yml` (ie, changing organizationName, organizationDomainName, superAdminCn)

```
$ grep -v "^#" group_vars/applicationServers.yml
---

wildfly_version: 26.0.1.Final
wildfly_checksum: 08d1e420331d0b6ad6c36a4dd782a110152cabfa23439e6ecd9a7c4d50bffd01
wildfly_standalone_file: standalone-26.xml.datasource.j2
wildfly_use_galleon: true
wildfly_galleon_version: 4.2.8
wildfly_galleon_checksum: sha256:63ccfb1546fbe32a2222b61e9ae0a594be5f81a4d6e45cd22861eb158a5bfb06
wildfly_jboss_client_checksum: sha256:859e69844bf047193f4ab461c7cc2b7c4b18ec76d1f32ebebcb9470097ef7dcd # WF 26.0.1
wildfly_disable_infinispan: true

wildfly_manage_service: true

wildfly_elytron_pass_dir: /opt/elytron
wildfly_elytron_pass_file: "{{ wildfly_elytron_pass_dir }}/wildfly_pass"
```

**Comment by svenska-primekey** [comment]
You have a connectivity issue that is preventing Galleon from downloading the wildfly pieces to install. You should try installing wildfly with the archive package instead of Galleon until you can resolve the network issue preventing Galleon from installing wildfly.

**Comment by mjourdan** [comment]
Calling the install script from the shell, I got this:

```
# /opt/galleon-4.2.8.Final/bin/galleon.sh install wildfly:current#26.0.1.Final --dir=/opt/wildfly-26.0.1.Final --default-configs=standalone/standalone.xml --layers=cloud-server,deployment-scanner,ejb,io,jsf,mail,webservices,-jsonb,-jms-activemq,-observability,-jmx-remoting,core-management,core-tools,managementnResolving feature-pack wildfly:current#26.0.1.FinalError: Install failed.
 * Failed to resolve Maven universe org.jboss.universe:community-universe
 * Failed to determine the latest version of org.jboss.universe:community-universe:jar:[0.0,). Make sure the remote Maven repository is properly configured and available.
```

After authorizing direct access to the web, this step could finish without error (even though proxy was still set in the environment).

It seems [galleon just ignores the proxy settings](https://developer.jboss.org/thread/280436n), and even though this has been reported to be [solved in Galleon 4.0.4](https://issues.redhat.com/browse/GAL-285), it still exists in Galleon 4.2.8.
